styles.css

```diff
@@ -20,11 +20,16 @@
   margin: 0;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
 body {
   display: flex;
```

Comment:
- [ ] It would be better if we avoid repeating CSS rules, in order to achieve the DRY principle, here is what we can do to fix the repeating rules; for example; display: flex and flex-direction: column has been repeated on the elements. we can create a class for it `.flex-column { display: flex; flex-direction: column }` or there are other ways you can explore as well